### ISSUE TITLE
React-Native (1.1.0) - border-radius no longer working

### ISSUE THREAD

**Issue by sync** [styled-components/styled-components#231]
## Version
1.1.0 (React Native 0.37.0) - yarn 0.17.6

## Reproduction
```
const Avatar = styled.Image`
  width: ${units.tabBarItem};
  height: ${units.tabBarItem};
  border-radius: ${units.tabBarItem / 2};
  background-color: ${colors.neutral50};
`;
```

## Expected Behavior
The avatar component to have rounded corners, was working in `1.0.1`

## Actual Behavior
The avatar component has no rounded corners

**Comment by mxstbr** [comment]
Ah damn, that's horrible. @jacobp100 any ideas why this regression happened?

**Comment by jacobp100** [comment]
Yep! https://github.com/styled-components/css-to-react-native/pull/9 . Sorry!

**Comment by mxstbr** [comment]
This should be fixed in `css-to-react-native@1.0.3`, run `npm install` again and you should be fine! Sorry for the troubles 🙏
